manifests/init.pp

```diff
@@ -247,6 +249,18 @@
     }
   }
 
+  case $::operatingsystem {
+    'Ubuntu': {
+      if $lsbmajdistrelease >= 14 {
```

Comment:
Are we 100% certain that the `lsb*` facts are available on all Ubuntu systems?

Perhaps @adrienthebo has a good estimation.